redash/tasks/queries.py

```diff
@@ -182,20 +183,28 @@ def refresh_queries():
     with statsd_client.timer('manager.outdated_queries_lookup'):
         for query in models.Query.outdated_queries():
             if settings.FEATURE_DISABLE_REFRESH_QUERIES:
-                logging.info("Disabled refresh queries.")
+                logging.debug("Disabled refresh queries.")
```

Comment:
This shouldn't be a debug message, because it low frequency and important.

```suggestion
                logging.info("Disabled refresh queries.")
```